.github/workflows/build.yml

```diff
@@ -200,6 +200,10 @@ jobs:
         run: python3 -m pip install -r requirements-docs.txt
       - name: Install (py)
         run: doit dev:py:jupyterlite-core
+      - name: Build the JavaScript kernel
+        run: |
+          yarn
```

Comment:
this is where caching the in-tree `node_modules` can save 30s-1min, depending on the os.

but also: should have doit task.

Reply:
> but also: should have doit task.

Not sure I'll have the bandwith and energy for this right now.

Also as mentioned before some doit tasks might or might not be executed depending on outside environment variable, which makes it very difficult to work with. Normally there should already be a task for building it already (should be `build:py:jupyterlite-javascript-kernel` I think) but it won't get triggered in the docs because of this environment variable.